test/unit/math/binaryOps/src/binaryOps.cpp

```diff
@@ -0,0 +1,432 @@
+//
+// Created by jakob on 19.08.19.
+//
+
+#include <catch2/catch.hpp>
+
+#include <alpaka/alpaka.hpp>
+#include <alpaka/test/MeasureKernelRunTime.hpp>
+#include <alpaka/test/acc/Acc.hpp>
+#include <alpaka/test/queue/Queue.hpp>
+
+#include <iostream>
+#include <typeinfo>
+#include <cmath>
+
+/*
+ * atan2     | Y | R^2\{(0,0)}
+ * fmod      | Y | R^2\{(x,0)|x in R}
+ * max       | Y | R^2
+ * min       | Y | R^2
+ * remainder | Y | R^2\{(x,0)|x in R}
+ * sincos    | Y | R^2 -> return 2 R
```

Comment:
since you are checking `sincos` now in this math unit tests we should remove the `sincos` unit test which we already have.